tests/_test_msui/test_remotesensing.py

```diff
@@ -56,3 +292,61 @@ def test_view_angles(self):
         angle = compute_view_angles(0, 0, 0, 0, -1, 0, 0, -1)
         assert angle[0] == 180.0
         assert angle[1] == -1
+
+    def test_body_angle(self):
+        compute_body_angle = RemoteSensingControlWidget(view=self.view).compute_body_angle
+        angle = compute_body_angle("sun", 73.56, 78.01, 25.27)
+        assert round(angle[0], 2) == 106.71
```

Comment:
Please use pytest approx
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx

Reply:
I'd also suggest to use pytest.mark.parametrize so separate the different tests:
https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#parametrize